### ISSUE TITLE
scaled scrolling

### ISSUE THREAD

**Issue by vshesh** [deltaepsilon/angular-markdown-editable#2]
I'm trying to use this directive for displaying 1-2 page long markdown documents (sort of a hack contenteditable markdown editor) and normally the markdown input will be longer than the corresponding rendered html. 

As such, I was wondering if you could implement some kind of scaled scrolling so that when I click on the box and the markdown source pops up, I'm closer to the actual source markdown for what I was viewing. Not sure how difficult this is, but I imagine you would just need to add a conversion based on client height of both elements (the hidden input and the shown view).

**Comment by deltaepsilon** [comment]
Yeah... this bothers me too, but I have no idea on where to start to get this implemented, and it's not a huge priority for me right now. Let's leave this issue open and maybe I'll take a crack at it some day?
